index.html

```diff
@@ -71,12 +79,132 @@ <h2>LET'S CONNECT</h2>
             </li>
           </ul>
     </section>
+    </section>
 
     <!-- headline ends -->
 
     <!-- Work section starts -->
 
     <section class="works">
+      <section class="cards-desktop">
+        <header class="project-snippet-desktop">
+          <a class="snippet-desktop" href="#"><img src="./Properties/snippet-desktop-one.png" alt=""></a>
+        </header>
+        <footer class="project-details-desktop">
+          <section class="project-details-heading-desktop">
+            <h2>Tonic</h2>
+            <ul>
+              <li class="first-list-item-desktop">CANOPY</li>
+              <li class="list-items-desktop">Back End Dev</li>
+              <li class="list-items-desktop">2015</li>
+            </ul>
+          </section>
+          <p>A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+         </p>
+         <ul class="tools-desktop">
+          <li class="tools-items-desktop">html</li>
+          <li class="tools-items-desktop">css</li>
+          <li class="tools-items-desktop">javascript</li>
+         </ul>
+         <button class="user-buttons-desktop">
+          See Project
+         </button>
+        </footer>
+      </section>
+
+      <section class="cards-desktop-second">
+        <header class="project-details-desktop-second">
+          <section class="project-details-heading-desktop-second">
+            <h2>Multi-Post Stories</h2>
+            <ul>
+              <li class="first-list-item-desktop-second">FACEBOOK</li>
+              <li class="list-items-desktop-second">Full Stack Dev</li>
+              <li class="list-items-desktop-second">2015</li>
+            </ul>
+          </section>
+          <p>
+            Experimental content creation feature that allows users to add
+            to an existing story over the course of a day without spamming
+            their friends.
+         </p>
+         <ul class="tools-desktop-second">
+          <li class="tools-items-desktop-second">html</li>
+          <li class="tools-items-desktop-second">Ruby on rails</li>
+          <li class="tools-items-desktop-second">css</li>
+          <li class="tools-items-desktop-second">javascript</li>
+         </ul>
+         <button class="user-buttons-desktop-second">
+           See Project
+         </button>
```

Comment:
- [ ] **[OPTIONAL]** Could you please review the **radius and border** of your buttons? It might be helpful to decrease them slightly to ensure they match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=QfxQv9P39nTqAsxx-0). This can help create a more consistent and cohesive design. 💯

### Your button:
![image](https://user-images.githubusercontent.com/107952724/236290524-184978f8-e4b8-4549-9e70-1662574d572f.png)

### Figma button:
![image](https://user-images.githubusercontent.com/107952724/236290572-f9d3970f-d849-45fd-ab8d-0cd2b0fd9814.png)